monai/data/synthetic.py

```diff
@@ -40,10 +40,16 @@ def create_test_image_2d(width, height, num_objs=12, rad_max=30, noise_max=0.0,
     norm = np.random.uniform(0, num_seg_classes * noise_max, size=image.shape)
     noisyimage = rescale_array(np.maximum(image, norm))
 
+    if channel_dim is not None:
+        assert isinstance(channel_dim, int) and channel_dim in (-1, 0, 2), 'invalid channel dim.'
+        noisyimage, labels = noisyimage[None], labels[None] \
```

Comment:
Is the point of channel_dim to create channels_first or channels_last image? If yes, then channel_dim int variable is not needed (It's not being used either). Can just have a boolean flag. Also safer for higher dimensional spatial data.

Reply:
It's not a bool, can have 3 types: None, channel first, channel last.
If channel_dim = None, will create an image without channel dimension.
Thanks.